sympy/utilities/runtests.py

```diff
@@ -507,6 +507,10 @@ def _test(*paths, **kwargs):
                     matched.append(f)
                     break
 
+    if slow:
+        random.seed(18042102)
```

Comment:
I think we need to figure out how to set it and then set it back, because the tests themselves print the seed (for further debugging) and my understanding is that they set it randomly. So we need to preserve that feature.

cc @asmeurer